domains/jamied132.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "JamieD132",
+    "email": "cheese@jamied132.is-a.dev",
+    "darflen": "JamieD132"
+  },
+  "record": {
+    "CNAME": "jamied132.github.io"
```

Comment:
I don't think you can add 2 domains in a single repo (if your goal is to use your site along with the is-a.dev one)

Reply:
I was planning on connecting my is-a.dev one to cloudflare pages instead and this one to github

Reply:
oh, should be fine then